timely_beliefs/docs/init.md

```diff
@@ -0,0 +1,91 @@
+    >>> bdf = tb.BeliefsDataFrame(s, belief_horizon=timedelta(hours=0), source=source, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-04 09:00:00+00:00 0 days         EPEX   0.5                              60
+
+## From a Pandas DataFrame
+
+Pass a Pandas DataFrame with columns ["event_start", "belief_time", "source", "cumulative_probability", "event_value"]. The "cumulative_probability" column is optional (a default of 0.5 will be used if the column is missing).
+
+    >>> df = pd.DataFrame([[63, datetime(2000, 1, 3, 9, tzinfo=pytz.utc), timedelta(hours=0), source], [60, datetime(2000, 1, 3, 10, tzinfo=pytz.utc), timedelta(hours=0), source]], columns=["event_value", "event_start", "belief_horizon", "source"])
+    >>> bdf = tb.BeliefsDataFrame(df, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-03 10:00:00+00:00 0 days         EPEX   0.5                              60
+
+Alternatively, keyword arguments can be used to replace columns that contain unique values for each belief.
+
+    >>> df = pd.DataFrame([[63, datetime(2000, 1, 3, 9, tzinfo=pytz.utc)], [60, datetime(2000, 1, 3, 10, tzinfo=pytz.utc)]], columns=["event_value", "event_start"])
+    >>> bdf = tb.BeliefsDataFrame(df, belief_horizon=timedelta(hours=0), source=source, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-03 10:00:00+00:00 0 days         EPEX   0.5                              60
+
+## From a CSV file
+
+The utility function `tb.read_csv` lets you load a BeliefsDataFrame from a csv file (e.g. example/temperature.csv).
+You still need to set the source and sensor for the BeliefsDataFrame; the csv file only contains their names.
+In case the csv file contains multiple source names, you can pass a list of sources.
+Each source name will then be replaced by the actual `Source` object from the list you provided.
+To write a BeliefsDataFrame to a csv file, just use the pandas way:
+
+    >>> bdf.to_csv("data.csv")
+    >>> tb.read_csv("data.csv", source=source, sensor=sensor)
+
+## From a list of TimedBeliefs
+
+Create a list of `TimedBelief` or `DBTimedBelief` objects and use it to initialize a BeliefsDataFrame.
+
+_This method is meant for applications with a database._
```

Comment:
Why is that? This example seems to work well without a DB. Do you mean that the other methods are otherwise preferrable?

Reply:
You're right, this is an acceptable method also without a database. I'll remove the comment.